lib/open_project/translations/models/locales_updater.rb

```diff
@@ -0,0 +1,189 @@
+# todo those ought to be in the gemspec
+require 'tempfile'
+require 'crowdin-api'
+require 'yaml'
+require 'zip'
+require 'pathname'
+require 'fileutils'
+
+require_relative '../helpers/tmp_directory'
+require_relative './git_repository'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.